common/utils/text_extractors.py

```diff
@@ -38,10 +93,11 @@ def __init__(self):
             '.jpg': 'image/jpeg'
         }
 
-    async def _process_file_async(self, file_path, folder_path_obj, graphname):
+    async def _process_file_async(self, file_path, folder_path_obj, graphname, temp_folder):
```

Comment:
@prinskumar-tigergraph can you add description for the args as some of them are generic names and unclear what's the real value it will be, e.g., file_path.